seeder/src/main/workers/background.ts

```diff
@@ -0,0 +1,17 @@
+import { parentPort, workerData } from "node:worker_threads";
+import { Shiinobi as _Shiinobi } from "$interfaces/shiinobi";
+
+const port = parentPort;
+if (!port) throw new Error("IllegalState");
+
+const shiinobi = new _Shiinobi();
+
+port.on("message", () => {
+	const _port = workerData.port;
+	try {
+		// @ts-ignore
+		const cmd = shiinobi.get_myanimelist_staff_urls();
```

Comment:
There are 2 issues here.

See if you can  add signatures.

Parse data nd fetch to express